odinclient/src/main/kotlin/me/odinclient/features/impl/dungeon/AutoBoom.kt

```diff
@@ -0,0 +1,58 @@
+package me.odinclient.features.impl.dungeon
+
+import me.odinmain.features.Category
+import me.odinmain.features.Module
+import me.odinmain.features.settings.impl.BooleanSetting
+import me.odinmain.features.settings.impl.NumberSetting
+import me.odinmain.utils.skyblock.*
+import me.odinmain.utils.runIn
+import me.odinclient.utils.skyblock.PlayerUtils
+import net.minecraft.client.Minecraft
+import net.minecraft.util.BlockPos
+import net.minecraft.util.ChatComponentText
+import net.minecraft.block.Block
+import net.minecraft.block.BlockStoneBrick
+import net.minecraftforge.event.entity.player.PlayerInteractEvent
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+
+object AutoBoom : Module(
+    name = "Auto Superboom",
+    description = "Places a superboom when you left-click.",
+    category = Category.DUNGEON,
+    tag = TagType.RISKY
+) {
+    private val minecraft: Minecraft = Minecraft.getMinecraft()
```

Comment:
Do not declare this again there is already a global variable for this
Don't copy paste random stuff into this mod